pkg/apis/numaplane/v1alpha1/isbservicerollout_types.go

```diff
@@ -36,7 +36,8 @@ type InterStepBufferService struct {
 
 // ISBServiceRolloutStatus defines the observed state of ISBServiceRollout
 type ISBServiceRolloutStatus struct {
-	Status `json:",inline"`
+	Status      `json:",inline"`
+	PauseStatus `json:"pauseRequestStatus,omitempty"`
```

Comment:
it appears that `PauseStatus` is inline? I think I'd prefer that it not be. Can you change it to 
`PauseRequestStatus PauseStatus `json:"pauseRequestStatus,omitempty"`?

Reply:
I didn't catch that the `PipelineRolloutStatus.PauseStatus` is the same - can that be changed as well?